portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,74 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+
+	var dropdown = document.getElementById("<%= namespace %>");
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	function onButtonKeyDown(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+			event.preventDefault();
+			list.classList.add('show');
+			button.setAttribute('aria-expanded', 'true');
+			var selectedOption = null;
+
+			Array.from(options).find(function(option) {
+				if (option.id == selectedItemValue) {
+					selectedOption = option;
+				}
+			});
+
+			if (selectedOption) {
+				selectedOption.focus();
+			}
+		}
+	}
+
+	button.addEventListener('keydown',onButtonKeyDown );
+
+	function handleKeyEvents(event) {
+		var currentIndex = Array.from(options).indexOf(document.activeElement);
+
+		if (event.key === 'ArrowDown') {
+			event.preventDefault();
+			if (currentIndex < options.length - 1) {
+				options[currentIndex + 1].focus();
+			}
+		} else if (event.key === 'ArrowUp') {
+			event.preventDefault();
+			if (currentIndex > 0) {
+				options[currentIndex - 1].focus();
+			}
+		}
+	}
+
+	list.addEventListener('keydown', handleKeyEvents);
+
+	function dropdownFocusOut(event) {
+		if (!dropdown.contains(event.relatedTarget)) {
+			list.classList.remove('show');
+			button.setAttribute('aria-expanded', 'false');
+		}
+	}
+
+	document.addEventListener('focusout', dropdownFocusOut );
+
+	var destroyDropDownPagination = function () {
+		button.removeEventListener('keydown', onButtonKeyDown);
+		document.removeEventListener('focusout', dropdownFocusOut );
+		list.removeEventListener('keydown', handleKeyEvents);
+
+		Liferay.detach('beforeScreenFlip', destroyDropDownPagination);
+	};
+
+	Liferay.on('beforeScreenFlip', destroyDropDownPagination);
```

Comment:
This is fine, but you could also do Liferay.once so that you don't have to detach the event yourself.